doc/developers/maintainer.rst

```diff
@@ -17,25 +36,70 @@ Before a release
    change log is reasonably well curated. Some tools for these tasks include:
 
    - ``maint_tools/sort_whats_new.py`` can put what's new entries into
-     sections.
+     sections. It's not perfect, and requires manual checking of the changes.
+     If the whats new list if well curated, it may not be necessary.
 
    - The ``maint_tools/whats_missing.sh`` script may be used to identify pull
      requests that were merged but likely missing from What's New.
 
-Preparing a bug-fix-release
-...........................
+Permissions
+...........
+
+The release manager requires a set of permissions on top of the usual
+permissions given to maintainers, which includes:
+
+- *maintainer* role on ``scikit-learn`` projects on ``pypi.org`` and
+  ``test.pypi.org``, separately.
+- become a member of the *scikit-learn* team on conda-forge.
+- *maintainer* on ``https://github.com/MacPython/scikit-learn-wheels``
+
 
-Since any commits to a released branch (e.g. 0.999.X) will automatically update
-the web site documentation, it is best to develop a bug-fix release with a pull
+.. _preparing-a-release-pr:
+
+Preparing a release PR
+......................
+
+Since any commits to an existing branch (e.g. 0.999.X) will automatically
+update the web site documentation, it is best to develop a release with a pull
 request in which 0.999.X is the base. It also allows you to keep track of any
-tasks towards release with a TO DO list.
+tasks towards release with a TO DO list. Since the documentation for the branch
+is updated live, it should ideally match the version available to users. Having
+all the changes for a release in one PR minimizes the gap between the time when
+the website points to a release, and the release being available to users.
 
-Most development of the bug fix release, and its documentation, should
-happen in master to avoid asynchrony. To select commits from master for use in
-the bug fix (version 0.999.3), you can use::
+Most development of the release, and its documentation, should happen in master
+to avoid asynchrony. First, create a branch, **on your own fork** (to release
+e.g. `0.999.3`)::
 
+    $ # assuming master and upstream/master are the same
     $ git checkout -b release-0.999.3 master
-    $ git rebase -i 0.999.X
+
+Note that pushing the (empty) branch **to the main repo** will automatically
+upload its documentation, which currently corresponds to that of master, so
+it's not an accurate representation of what the release will actually be.
+
+Then create a PR **to the `scikit-learn/0.999.X` branch and not `master`** with
+all the desired changes::
+
+	$ git rebase -i upstream/0.999.2
```

Comment:
So this is this a branch in the fork or in the official scikit-learn repo? The documentation doesn't say how it was created..

Reply:
I added a step 0